tests/e2e/utils/auto-rds-s3-cleanup.py

```diff
@@ -0,0 +1,86 @@
+import argparse
```

Comment:
this script is missing deleting the secrets manager driver and service accounts created by setup script